test/evaluation/inputDescriptorFilterEvaluationHandler.spec.ts

```diff
@@ -29,7 +27,14 @@ describe('inputDescriptorFilterEvaluationHandler tests', () => {
 
   it(`input descriptor's constraint property missing`, () => {
     const inputCandidates: unknown = getFile('./test/dif_pe_examples/vp/vp_general.json');
-    const presentation: Presentation = new Presentation(inputCandidates['@context'], inputCandidates['presentation_submission'], inputCandidates['type'], inputCandidates['verifiableCredential'], inputCandidates['holder'], inputCandidates['proof']);
+    const presentation: VerifiablePresentation = {
+      '@context': inputCandidates['@context'],
+      presentationSubmission: inputCandidates['presentation_submission'],
+      type: inputCandidates['type'],
+      verifiableCredential: inputCandidates['verifiableCredential'],
+      holder: inputCandidates['holder'],
+      proof: inputCandidates['proof']
+    };
```

Comment:
insantiation should be with new , that is cleaner

Reply:
Nope it is not. You are looking at this with a Java hat on